mhs/outbound/outbound/transmission/outbound_transmission.py

```diff
@@ -25,18 +25,21 @@ class OutboundTransmission(transmission_adaptor.TransmissionAdaptor):
 
     """A component that sends HTTP requests to a remote MHS."""
 
-    def __init__(self, certs_dir: str, client_cert: str, client_key: str, ca_certs: str, max_retries: int):
+    def __init__(self, certs_dir: str, client_cert: str, client_key: str, ca_certs: str, max_retries: int,
+                 retry_delay: int):
         """Create a new OutboundTransmission that loads certificates from the specified directory.
         :param certs_dir: A string containing the path to the directory to load certificates from.
         :param client_cert: A string containing the name of the client certificate file in the certs directory.
         :param client_key: A string containing the name of the client private key file in the certs directory.
         :param ca_certs: A string containing the name of the certificate authority certificate file in the certs directory.
         :param max_retries: An integer with the value of the max number times to retry sending the request.
+        :param retry_delay: An integer representing the delay (in milliseconds) to use between retry attempts.
```

Comment:
The number of parameters here seems to be getting a bit long to me, I think it's probably fine for the time being but I think if there's ever another parameter added or we get time for a refactor it's probably worth splitting these into their own certs/retry data objects